### DISCUSSION TITLE
SSL website not working

### DISCUSSION THREAD

**Discussion by DanIsNearby** [requarks/wiki#6143] [Error / Bug Report] [answered]
I have downloaded the wiki.js using the Linux guide configured everything and generated SSL on cloudflare using .pem after launching the website opening ports and adding firewall http works but on the port 3443 i get 

"The connection was reset

The connection to the server was reset while the page was loading.

    The site could be temporarily unavailable or too busy. Try again in a few moments.
    If you are unable to load any pages, check your computer’s network connection.
    If your computer or network is protected by a firewall or proxy, make sure that Firefox is permitted to access the web."

**Comment by NGPixel** [comment]
You should map the 3443 port to 443. 3443 is the internal port only.

**Reply by DanIsNearby** [reply]
sudo firewall-cmd --add-forward-port=port=443:proto=tcp:toport=3443 

right? or the other way around?

**Comment by DanIsNearby** [comment, marked as answer]
turns out my SSL worked with the way i did it i just didnt give it enought time to be asigned 🤦‍♂️
